zonemaster/zonemaster-gui
# Fix favicon images

## Purpose

Fix the favicons size

## Context

Fix #119

## Changes

Regenerate all favicons so that the logo is not distorted.

## How to test this PR
The logo should have the correct ratio size in all those cases : (I can not test on max / ios)

* Load the GUI, check the favicon
* Add the website to the visited sites / shortcuts in firefox
* Add the website to the visited sites / shortcuts in chrome
* Add the website to the windows 10 start new
* Add the website to the android home screen
